feature/system/tests/system_base_test/g_protocol_test.go

```diff
@@ -58,8 +59,14 @@ func TestGNMIClient(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	conn := dialConn(t, dut, introspect.GNMI, 9339)
 	c := gpb.NewGNMIClient(conn)
-	if _, err := c.Get(context.Background(), &gpb.GetRequest{Encoding: gpb.Encoding_JSON_IETF, Path: []*gpb.Path{{Elem: []*gpb.PathElem{}}}}); err != nil {
-		t.Fatalf("gnmi.Get failed: %v", err)
+	if deviations.GnmiGetRequiresConfigType(dut) {
```

Comment:
Hi robshakie,
As per the comment suggested by @dplore this change was made with the deviation request-
https://github.com/openconfig/featureprofiles/pull/3265#issuecomment-2229646487

Please let me know if any other changes needs to be done.

Thanks,
C Akhil

Reply:
@robshakir If seems Junos only supports gNMI.Get on config leaves.  This is not compliant with gnmi, but we also want to be aware of this limitation.

I recommend that we accept a deviation for this because this is not blocking operational requirements and allows us to document the issue.   Juniper should remove this deviation on a future release of Junos. 

Note that IOSXR has a related but different issue; gNMI.Subscribe is not supported on config leaves.

Reply:
What is our plan to resolve this deviation? Please ping me internally with this information. I think this is proliferating an ambiguity in requirements. No change to my blocking comments.